src/connect/connect.cpp

```diff
@@ -1,9 +1,9 @@
 #include "connect.hpp"
-#include "httpc.hpp"
+#include "http/httpc.hpp"
```

Comment:
These should (and the below one too) should become `#include <http/httpc.hpp>`, as it is path include now, not a relative one. Also, it's worth moving to the relevant group with includes from other our directories below.